### ISSUE TITLE
Oddy chem dupe

### ISSUE THREAD

**Issue by Pyko1** [HippieStationCode/HippieStation13#2991]
Can be an issue as well.

**Comment by banthebantz** [comment]
Yeah sure

**Comment by ArcLumin** [comment]
you can dupe with cryo and smoke too though

**Comment by banthebantz** [comment]
Cryo dupe is slower and requires a multi step process to extract the product, smoke is the same since it dupes inside mobs so you have to gib them or induce a shit ton of vomiting to extract it.

**Comment by Pyko1** [comment]
In my opinion only the most basic chems should be synthable, or make it take a much more time and charge to push out complex shit really.

Currently as it is, oddy is runing nice things, as we have to account it in.

**Comment by banthebantz** [comment]
I have a compromise, blacklist chems with the basic synthesizer and add an advanced one as a researchable upgrade.

**Comment by Pyko1** [comment]
Or maybe even have modules/upgrades to add chems to choose from?
Would be a lot easier to balance.

Ie. 
Level 1 = basic healing chems for 50 charge/u
Level 2 = Morphine for 100 charge/u
level 3 = Chloral for 200 charge/u
level 4 = shit for even more higher charge price.

Level illegal = (emag or maybe traitor upgrade) real bad ass shit for heavy price.

**Comment by Pyko1** [comment]
as it is oddy is a full fledged combat mech tanks to chem duper, and piss easy to make.

**Comment by ArcLumin** [comment]
It's really weak defensively, but it's also one of the faster mechs

**Comment by Pyko1** [comment]
yeah but you can pretty much make it at round start, and you have snowballs chance of hell to win against it if it jumps on you.

**Comment by ArcLumin** [comment]
Unless you have a hardsuit

**Comment by trifornt-tinfornt** [comment]
Gonna be honest, I never really thought the Odysseus was more powerful than a traitor running around killing people with ebow esword or something. Odysseus might be able to practically guarantee a kill, but due to the chem recharging, hardsuits, etc. it's a rather slow killer that takes time between kills - not to mention the recharge. Plus if Sec is remotely competent they can just grab an Ion Gun, jump in a hardsuit and charge you then you're right fucked unless you grabbed other weaponry.

Dunno why it should be nerfed. I've seen easier ways to kill people in seconds - and ones that don't even require construction.

**Comment by ArcLumin** [comment]
Yeah, only thing about it, it's a slow and VERY obvious killer with little else in the form of defences, completely fucking walled by hardsuits

**Comment by LufgtHuron** [comment]
When the fuck have you ever seen an oddyseus gunning down crew with traitor chems? This literally **never** happens. No one even complains about it. Because no one ever does this. Its not dangerous. Its not robust. Shit, its not even clever.

Defeated by hardsuits, ions and pretty much anything else when attacked by a robust player. Very limited ammo, when it runs out of syringes it can't do anything. Noisy and very high profile. 99% of times something will go wrong and you won't get the shit from science/cargo to even make it. This was never good and it was so MUCH WORSE on Hippie. 

All these "nerfs" will just run it completely into shit so whenever someone actually bothers go through the hell to get this they will be completely disappointed. And no one on Hippie will ever make an oddyseus again.

The levels are just ridiculous considering why the fuck would I build an oddyseus to make chloral when I can make a chem dispenser and make it in few clicks much faster with plenty of other options?

Duplicator in the oddy works solely how it is now. Changing it is to completely destroy it and no, I don't care you got killed by this in that one round out of a thousand.

**Comment by ArcLumin** [comment]
The most robust thing an oddy is used for is to dupe chems for other usages or to chase down shadowlings with their high speed and light they can't shut off

**Comment by McBawbaggings** [comment]
https://www.youtube.com/watch?v=KxXHA-bPmjI

It happens sometimes.

**Comment by banthebantz** [comment]
Well my main issue with it is how you can abuse shooting syringes at walls to dupe any chem and get it in a beaker, fix that and I'll be happy

**Comment by trifornt-tinfornt** [comment]
Yeah, syringes that land should probably just spill on the ground as water or something. Other than that bug, the Odysseus has no real issues with it.
